Katello/katello
# Fixes #21493 - Don't blindly regen certs on manifest refresh

This is effectively a revert of https://github.com/Katello/katello/pull/6641 which is needed because Candlepin 2.0 will do this for us automatically and more intelligently.

To test:

- Refresh a Manifest which you have imported prior to checking out this PR
- Find the related Task and view the dynflow console
- Confirm that Actions::Candlepin::Owner::UpstreamRegenerateCertificates is not executed from the Run tab
